### ISSUE TITLE
clustering

### ISSUE THREAD

**Issue by debonair** [e2ebridge/bpmn#36]
I am having trouble clustering the bpmn using pm2. I can only find process that were created by that node of the cluster. is it possible to refresh the process cache from what has been persisted to the data store?

**Comment by TomKaltz** [comment]
I'm curious about this too @debonair do you have any new findings?

**Comment by wiltshirek** [comment]
Yeah, we had the same issue a while back. Had to go with sticky LB sessions
for the mongodb connection.

On Mon, Jul 4, 2016 at 9:33 AM, Duma Mtungwa <notifications@github.com>
wrote:

> I am having trouble clustering the bpmn using pm2. I can only find process
> that were created by that node of the cluster. is it possible to refresh
> the process cache from what has been persisted to the data store?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/e2ebridge/bpmn/issues/36>, or mute the thread
> <https://github.com/notifications/unsubscribe/AE3y74sO-TcHO_gWHX3qrodCV7mb2W3kks5qSQuNgaJpZM4JEX9l>
> .
>



-- 
*Ken Wiltshire** |* Computer Applications Developer
5715 Ave N *|* Brooklyn, NY 11234

Direct (347) 675-5213

**Comment by wiltshirek** [comment]
More info here.  The sticky solution works for that current session only.
If your design calls for updates that can cross sessions (which is going to
be the case in most instances by nature) then you'll have to go with one
mongodb instance.

I guess you could use two mongo instances with sticky sessions and mongo
replication to avoid having a single point of failure.

On Fri, Aug 5, 2016 at 11:26 AM, Ken Wiltshire <wiltshirek@gmail.com> wrote:

> Yeah, we had the same issue a while back. Had to go with sticky LB
> sessions for the mongodb connection.
>
> On Mon, Jul 4, 2016 at 9:33 AM, Duma Mtungwa <notifications@github.com>
> wrote:
>
>> I am having trouble clustering the bpmn using pm2. I can only find
>> process that were created by that node of the cluster. is it possible to
>> refresh the process cache from what has been persisted to the data store?
>>
>> —
>> You are receiving this because you are subscribed to this thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/e2ebridge/bpmn/issues/36>, or mute the thread
>> <https://github.com/notifications/unsubscribe/AE3y74sO-TcHO_gWHX3qrodCV7mb2W3kks5qSQuNgaJpZM4JEX9l>
>> .
>>
>
>
>
> --
> *Ken Wiltshire** |* Computer Applications Developer
> 5715 Ave N *|* Brooklyn, NY 11234
>
> Direct (347) 675-5213
>
>
>



-- 
*Ken Wiltshire** |* Computer Applications Developer
5715 Ave N *|* Brooklyn, NY 11234

Direct (347) 675-5213
